backend/src/main/kotlin/app/ehrenamtskarte/backend/application/webservice/schema/create/WorkAtOrganization.kt

```diff
@@ -0,0 +1,35 @@
+package app.ehrenamtskarte.backend.application.webservice.schema.create
+
+import app.ehrenamtskarte.backend.application.webservice.schema.view.AttachmentView
+import app.ehrenamtskarte.backend.application.webservice.schema.view.JsonField
+import app.ehrenamtskarte.backend.application.webservice.schema.view.Type
+import app.ehrenamtskarte.backend.application.webservice.utils.JsonFieldSerializable
+
+enum class AmountOfWorkUnit {
+    HOURS_PER_WEEK,
+    HOURS_PER_YEAR
+}
+
+data class WorkAtOrganization(
+    val organization: Organization,
+    val amountOfWork: Double,
+    val amountOfWorkUnit: AmountOfWorkUnit,
+    val certificate: Attachment
```

Comment:
I would make the certificate optional because the confirmation by the organization could also be done by phone call for example